src/masternodes/mn_checks.cpp

```diff
@@ -445,6 +445,15 @@ Res ApplyAccountToUtxosTx(CCustomCSView & mnview, CCoinsViewCache const & coins,
     if (msg.balances != *minted.val) {
         return Res::Err("%s: amount of minted tokens in UTXOs and metadata do not match: (%s) != (%s)", base, minted.val->ToString(), msg.balances.ToString());
     }
+
+    // block for non-DFI transactions
+    for (auto const & kv : minted.balances) {
+        DCT_ID tokenId = kv.first;
```

Comment:
Can use `const DCT_ID& tokenId = kv.first;` To avoid extra call of assign operator